common/djangoapps/student/tests/test_views.py

```diff
@@ -119,6 +120,29 @@ def test_cant_unenroll_status(self):
 
             self.assertEqual(response.status_code, 200)
 
+    def test_course_run_refund_status_successful(self):
+        """ Assert that the course refund status is 200 for successful refund call."""
+
+        with patch('student.models.CourseEnrollment.refundable', return_value=True):
+            response = self.client.get(reverse('course_run_refund_status', kwargs={'course_id': self.course.id}))
```

Comment:
Why don't we assert the response body is of `{"course_refundable_status": true}`?